chapters/packaging.Rmd

```diff
@@ -182,6 +183,10 @@ though,
 we should create a \gref{virtual environment}{virtual_environment}\index{virtual environment (in Python)}\index{Python!virtual environment}
 to test how our package installs
 without breaking anything in our main Python installation.
+We created an environment in Chapter \@ref(provenance)
```

Comment:
We exported the details of our environment in the provenance chapter, but we didn't create an environment in that chapter (i.e. we just ran `$ conda env export > environment.yml` from the base environment).

Reply:
Right! Updated the language here.